org.eclipse.lsp4e/src/org/eclipse/lsp4e/callhierarchy/CallHierarchyContentProvider.java

```diff
@@ -33,29 +35,31 @@
 import org.eclipse.lsp4j.CallHierarchyIncomingCallsParams;
 import org.eclipse.lsp4j.CallHierarchyItem;
 import org.eclipse.lsp4j.CallHierarchyPrepareParams;
+import org.eclipse.lsp4j.Position;
 import org.eclipse.lsp4j.Range;
 import org.eclipse.lsp4j.ServerCapabilities;
+import org.eclipse.lsp4j.TextDocumentIdentifier;
 import org.eclipse.ui.PlatformUI;
 
 /**
  * Content provider for the call hierarchy tree view.
  */
 public class CallHierarchyContentProvider implements ITreeContentProvider {
-	private TreeViewer treeViewer;
-	private LanguageServerWrapper languageServerWrapper;
-	private List<CallHierarchyViewTreeNode> rootItems;
+	private @Nullable TreeViewer treeViewer;
+	private @Nullable LanguageServerWrapper languageServerWrapper;
+	private @Nullable List<CallHierarchyViewTreeNode> rootItems;
 	private String rootMessage = Messages.CH_finding_callers;
 
 	@Override
-	public Object[] getElements(final Object inputElement) {
+	public Object[] getElements(final @Nullable Object inputElement) {
 		if (rootItems != null) {
 			return rootItems.toArray();
 		}
 		return new Object[] { rootMessage };
 	}
 
 	@Override
-	public Object[] getChildren(final Object parentElement) {
+	public Object [] getChildren(final Object parentElement) {
```

Comment:
```suggestion
	public Object[] getChildren(final Object parentElement) {
```
to make it consistent with the rest of the codebase